.travis.yml

```diff
@@ -6,7 +7,34 @@ python:
   - "3.3"
   - "3.4"
   - "3.5"
-  - "3.5-dev" # 3.5 development branch
-  - "nightly" # currently points to 3.6-dev
+  - "3.6"
+  - "nightly" # currently points to 3.8-dev
 # command to run tests
 script: nosetests
+
+jobs:
+  include:
+  - python: 3.7
+    dist: xenial
+    sudo: true
+
+  - stage: &stage_deploy_name upload new version of python package to PYPI (only for tagged commits)
+    name: "Deploy"
```

Comment:
Quotes are not really needed for strings unless you use some special characters there, which you don't.